CHANGELOG.md

```diff
@@ -0,0 +1,39 @@
+Change log
+===
+
+### Newer Versions 
+
+For newer versions. See [GitHub releases](https://github.com/jenkinsci/matlab-plugin/releases)
+
+### 1.0.3
+
+Release date: _Nov 05, 2019_
+
+* Bug Fix : Scratch file copy is restricted to "Automatic" option.
+* README doc updates. 
+
+
+### 1.0.2
+
+Release date: _Aug 21, 2019_
+
+* MATRIX build support. 
+* Plugin support for older versions of MATLAB till R2013a.
+
+### 1.0.1
+
+Release date: _May 10, 2019_
+
+* Jenkins run-time environment variables support in MATLAB script.
+* Enhancement of README with with SCM configuration step for "Automatic" Test mode.
+
+### 1.0.0
+
+Release date: _Apr 08, 2019_
+
+* Initial release 

```

Comment:
Please add a period to the end of the note to be consistent with other notes.

Initial release.

Reply:
Done.